src/main/kotlin/com/milaboratory/mixcr/basictypes/GeneFeatures.kt

```diff
@@ -85,7 +89,12 @@ data class GeneFeatures(
 
     companion object {
         @JvmStatic
-        fun parse(value: String): GeneFeatures = GeneFeatures(GeneFeature.parse(value))
+        fun parse(value: String): GeneFeatures = if (value.startsWith("[")) {
```

Comment:
you sad that you removed it

Reply:
I removed it before, when it was a JsonCreator.